platforms/netlify-ci-build/README.md

```diff
@@ -1,8 +1,17 @@
 # Introduction
 
-Prisma and Netlify (beta build) integration.
+Prisma and Netlify [beta build](https://build-beta.netlify.com/) integration.
 
 # Friction Points
 
 - https://github.com/netlify/build/issues/730
+- Not zero-config, binary target configuration needed
```

Comment:
Isn't this the same thing as the issue above? If so, these should be combined into 1 friction point I guess?

Reply:
:(

Reply:
Yes? No?

Reply:
:( is always a yes!